src/kernel/include/fpu.h

```diff
@@ -38,6 +38,7 @@ static inline int fpu_check_fxsr(void);
 static inline int
 fpu_get_info(void)
 {
+	/*
```

Comment:
going to move this to chal :) M7 have  double-precision FPU we can take advantage of.